hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/token/impl/CryptoServiceImpl.java

```diff
@@ -1,36 +0,0 @@
-/*
- * Copyright (C) 2020-2022 Hedera Hashgraph, LLC
```

Comment:
We should delete `StandardCryptoService` and keep this class ?

Reply:
We had a discussion some time ago that it won't be the best idea to call the implementations of our services `*Impl` since we could have more than 1 implementation. Based on that I decided to use `Standard*` as a name definition for the service implementations. I deleted the old dummy and renamed the `CryptoServiceImpl` to `StandardCryptoService`.

Reply:
Discussed again this morning. IIRC general agreement was that `impl` isn't great, but it's not offensive enough to change and there weren't any better suggestions.

Reply:
I created an issue for that https://github.com/hashgraph/hedera-services/issues/4478